src/gsy_e/models/strategy/__init__.py

```diff
@@ -48,12 +48,11 @@
 
 
 if TYPE_CHECKING:
-    from gsy_framework.data_classes import TradeBidOfferInfo
-
-    from gsy_e.models.market.one_sided import OneSidedMarket
-    from gsy_e.models.market.two_sided import TwoSidedMarket
+    from gsy_framework.data_classes import TradeBidOfferInfo  # noqa: F401
+    from gsy_e.models.market.one_sided import OneSidedMarket  # noqa: F401
+    from gsy_e.models.market.two_sided import TwoSidedMarket  # noqa: F401
 
-INF_ENERGY = int(sys.maxsize)
+INF_ENERGY = int(10000000)
```

Comment:
I was already surprised when I saw the offer from the Infinite bus that you shared. Can you please explain why you went with 10000000. Is this to better locate those offers in the logs?

Reply:
I removed it in order to avoid possible integer overflow errors. In theory, if we do this operation for example `1.5 * sys.maxsize` , the integer would overflow and the result of this operation would be `inf`. To exclude this possibility from being the cause of floating-point precision errors, I decided to use a much smaller integer (but still huge in this context, this is actually 10 GWh)